code/model/UserDefinedForm.php

```diff
@@ -319,7 +319,13 @@ public function init() {
 		Requirements::javascript(FRAMEWORK_DIR .'/thirdparty/jquery/jquery.js');
 		Requirements::javascript(USERFORMS_DIR . '/thirdparty/jquery-validate/jquery.validate.min.js');
 		Requirements::add_i18n_javascript(USERFORMS_DIR . '/javascript/lang');
-		Requirements::javascript(USERFORMS_DIR . '/javascript/UserForm.js');
+
+		// Don't load the form script when the form has been submitted.
+		// We don't want to try and re-initialise the form when it's hidden and can't be used.
+		if(Session::get('FormProcessed') == null) {
```

Comment:
Can you move this logic into the javascript itself? It needs to gracefully fall back if included, but the form itself is hidden (perhaps for some other reason).